<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="logtext">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>821</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Log</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>431</width>
      <height>301</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLCDNumber" name="value5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>250</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="description0">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>flow4</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="description3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Setpoints</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>flow0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>flow1</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="setpoint5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> ml/min</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="setpoint0">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> ml/min</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="setpoint2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> ml/min</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Actual values</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="setpoint4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> ml/min</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="description5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="value2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>130</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="setpoint3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> ml/min</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="value4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>210</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>flow5</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="setpoint1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="suffix">
      <string> ml/min</string>
     </property>
     <property name="maximum">
      <double>10.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QLineEdit" name="description2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>flow2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="description1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="value3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>170</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="value1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLCDNumber" name="value0">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
     <property name="smallDecimalPoint">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="description4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>66</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>flow3</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Flows&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_flow_file_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Load flow file</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="flow_file">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_flow_file_btn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start flow file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_flow_file_btn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop flow file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
